fe/fe-core/src/main/java/org/apache/doris/job/extensions/insert/InsertTask.java

```diff
@@ -38,24 +38,19 @@
 import com.google.common.base.Joiner;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.Lists;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.extern.log4j.Log4j2;
 
-import java.util.ArrayList;
-import java.util.List;
 import java.util.Optional;
 import java.util.UUID;
 import java.util.concurrent.atomic.AtomicBoolean;
 
-/**
- * todo implement this later
- */
 @Log4j2
 public class InsertTask extends AbstractTask {
 
     public static final ImmutableList<Column> SCHEMA = ImmutableList.of(
+            new Column("JobId", ScalarType.createStringType()),
```

Comment:
Placed under TaskId? Because MTMVTask, jobId is under taskId, try to maintain consistency as much as possible

Reply:
done